medusa/history.py

```diff
@@ -22,7 +22,7 @@
 import db
 from .common import FAILED, Quality, SNATCHED, SUBTITLED
 from .helper.encoding import ss
-from .show.History import History
+from .show.history import History
```

Comment:
@labrys i would import as `show_history` to not confuse with history.py. maybe also rename show/show_history.py